Marlin/src/lcd/dogm/status_screen_DOGM.cpp

```diff
@@ -388,19 +397,31 @@ void MarlinUI::draw_status_screen() {
       ;
       duration_t elapsed = print_job_timer.duration();
       const uint8_t p = progress & 0xFF, ev = elapsed.value & 0xFF;
-      if (progress > 1 && p != lastProgress) {
+      if (progress > 1 || p != lastProgress) {
```

Comment:
This kind of kills the `if (progress > 1)` condition, so it might as well be removed.